packages/web/app/routes/feed[.xml].tsx

```diff
@@ -26,11 +26,13 @@ export const loader = (async () => {
 
   const sortedAwesomeYasunori = sortOn(awesomeYasunori, "-date");
   for (const yasunori of sortedAwesomeYasunori) {
+    const link = `${domain}#${yasunori.id}`;
```

Comment:
アドレスとしては、本来これが正しいはず。

```suggestion
    const link = `${domain}/#${yasunori.id}`;
```

ただ、間違っていても（おそらく）ブラウザの機能で redirect されるみたい。

Reply:
なので LGTM with comments